### PR TITLE
Prep Master for 2.0.0 Release

### PR DESCRIPTION
**Pull request by CITguy**
JIRA: n/a

The `1.x-LTS` branch has been created to handle the remaining `1.50.x` releases (including tomorrow's `1.50.1`). This PR is to bring all 2.x work back into master (effectively bringing `master` up to v2.0.0-9), so we don't have to wait on Travis/CodeShip to release tomorrow.

### LGTMs
- [ ] Dev LGTM
- [ ] Dev+Vidyo LGTM
- [ ] QE LGTM

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,9 @@
 <a name="1.50.0-0"></a>
 ### 1.50.0-0 (2016-06-15)
 
+<a name="2.0.0-9"></a>
+### 2.0.0-9 (2016-06-15)
```
Comment (parlarjb) [inline CHANGELOG.md]: Is there a way to get a 2.0.0 entry into the change log?

File: bower.json [hunk 2]
```diff
@@ -1,28 +1,27 @@
 {
   "name": "encore-ui",
-  "version": "1.50.0",
+  "version": "2.0.0-9",
```
Comment (parlarjb) [inline bower.json]: What needs to be done to get this to 2.0.0?

File: guides/using-encoreui.md [hunk 3]
```diff
@@ -24,13 +24,13 @@ To use the CDN in your app, do the following:
 2. Include the EncoreUI CSS file in the `<head>` of your index.html (before the CSS for your app):
 
     ```
-    <link rel="stylesheet" href="https://95c7050854423f809e66-6999ba0e7a4f47d417515fb3f08fa9b8.ssl.cf1.rackcdn.com/encore-ui-1.50.0.min.css">
+    <link rel="stylesheet" href="https://95c7050854423f809e66-6999ba0e7a4f47d417515fb3f08fa9b8.ssl.cf1.rackcdn.com/encore-ui-2.0.0-9.min.css">
```
Comment (parlarjb) [inline guides/using-encoreui.md]: Same thing here as above. (I'll stop pointing these out for now)

File: bower.json (hunk 2 above)
Comment (CITguy) [inline bower.json]: that'll happen tomorrow with `grunt shipit:major`

File: CHANGELOG.md (hunk 1 above)
Comment (CITguy) [inline CHANGELOG.md]: 2.0.0 hasn't officially released. The `master` to `2.x` syncs have been really wonky wrt what info goes in which version, and we've been much more detailed with the GitHub releases, so CHANGELOG isn't of much use any more. Either we'll be removing or fixing CHANGELOG after 2.0.0 releases.

File: bower.json (hunk 2 above)
Comment (CITguy) [inline bower.json]: This PR is not to get it to v2.0.0, but put `master` in a good state to _release_ v2.0.0.

**Comment by CITguy** [comment]
I have to close this PR. I need to create a temporary branch between `2.x` and `master` to handle merging changes in both branches. I'll submit a new PR shortly.

**Pull request closed without merging**